### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by maherramadan78**
Create a website consisting of 3 pages: Home, Calculator, and Quote.
I used correct Gitflow.
The general layout is  matching  these wireframes:
Add my  own styles to improve the look and feel. 
Use React basic styling (inline styles, importing CSS and pre-processed CSS files).
the 3 navigation links for all routes are displayed on each page.
Use router to make the router works.

### CODE REVIEW

File: src/components/styles/home.module.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+}
+
+.h2Home {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-size: 2rem;
+}
+
+.homeDiv > p {
+  font-size: 30px;
+}
```
Comment (Tufahel) [inline src/components/styles/home.module.css]: - [ ] Kindly match the basic structure with the wireframe. The `Math Magicians` logo should be aligned with the `Welcome to our page` and other content of the page.

image

File: src/components/styles/Calculator.module.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  display: flex;
+  width: 100%;
+  height: 80vh;
+}
+
+.calculatorDiv > h2 {
+  font-size: 30px;
+  text-align: center;
+  width: 40%;
+}
```
Comment (Tufahel) [inline src/components/styles/Calculator.module.css]: - Kindly fix some issues below:
  - [ ] Kindly match the basic structure with the wireframe. The `Math Magicians` logo should be aligned with the `Let's do some math!` and other content of the page.
  - [ ] Kindly match the basic structure with the wireframe. The `Let's do some` text should be aligned with the `Calculator interface` horizontally.
  
  image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @maherramadan78 :wave: 

Well done. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ Commit messages are professional.
✔️ Gitflow is correct.
✔️ Readme is professional.


## Required Changes ♻️

> Kindly check the inline comments for review.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @maherramadan78, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**